microcom.h

```diff
@@ -74,7 +74,7 @@ extern struct ios_ops *ios;
 extern int debug;
 extern int opt_force;
 extern int listenonly;
-extern char *answerback;
+extern unsigned char *answerback;
```

Comment:
Ah, you change the type of answerback, this makes some of the feedback above wrong, but this doesn't fit into the commit's topic, does it?